pages/dkp/kommander/1.4/release-notes/index.md

```diff
@@ -22,14 +22,11 @@ To get started with Kommander, [download](/dkp/konvoy/latest/download/) and [ins
 Kommander provides a command center for all your cloud native management needs in public Information as a Service (IaaS), on-premises, and edge environments. Kommander provides a multi-tenant experience to create, secure, and configure Kubernetes clusters and cloud native workloads. Additionally, Kommander enables teams to unlock federated cost management across multiple clusters, whether they are a new Konvoy cluster or an existing 3rd party/DIY distribution installation.
 
 # Supported Versions
```

Comment:
We decided to remove this table from Kommander RN, right?  It's just too confusing because Konvoy also has a table and they get out of sync.